### DISCUSSION TITLE
Skip default value if the store exists?

### DISCUSSION THREAD

**Discussion by ctwhome** [joshnuss/svelte-local-storage-store#242] [Q&A] [answered]
Hi Joshua!

thanks for this amazing library! 

I am not sure if this is correct, before loading the store from localStorage, the default value is presented in the UI and then it changes to what it retrieves from the persisted store. Is there a way to avoid this behaviour?

![Feb-12-2024 13-41-45](https://github.com/joshnuss/svelte-persisted-store/assets/4195550/16141d82-d393-45a3-bc46-15c5fad8ddf4)

**Comment by joshnuss** [comment, marked as answer]
Hi @ctwhome!

Is your code rendering the store server side?

Since Local storage is only available in the browser, areas that depend on it should not be rendered server-side.

**Reply by ctwhome** [reply]
yes indeed! I thought I was already disabling it, but not. Now yes:

+layout.ts
```
// Disable server-side rendering
export const prerender = false
export const ssr = false;
```

Thanks a lot

**Reply by joshnuss** [reply]
Welcome!
